### PR TITLE
[🛠] Package update and asset regenerate

### PR DESCRIPTION
**Pull request by glasnt**
Should: 
 * resolve github banner notification, bumping `minimist` to >=1.2.2
 * minor bump of 329 other packages per auto-fix (326 low, 2 moderate)
 * re-compile assets accordingly

Does not:
 * resolve manual review for netlify-cli > gh-release-fetch > download > decompress 
   * ref netlify/cli#731

Created/tested locally with: 
 * `npm audit fix`
 * `gulp dev`
 * `hugo server -w ...`

And nothing seems terribly wrong.

The netlify preview will not be a complete test in this case. 

Deferring to @mattstratton as an uber-reviewer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,15 +26,16 @@
...
     "gulp-uglify": "^3.0.2",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.13.1",
     "require-dir": "~1.2.0"
   },
   "dependencies": {
-    "netlify-cli": "^2.15.0",
     "bootstrap": "^4.3.1",
     "font-awesome": "^4.7.0",
+    "fsevents": "^2.1.2",
```
Comment (mattstratton) [inline package.json]: this is what is making this break on the netlify build; do you know what pulled this in?

Comment (mattstratton) [reply package.json]: tl;dr is that this should be somehow seen as an optional dependency (it only works on MacOS, so it's bailing on the netlify part).

Basically, even though we aren't using any of it, netlify still installs everything in `package.json` when it builds the site (we cannot prevent this from happening) and the build containers aren't MacOS so ... failure.

**Comment by mattstratton** [comment]
For troubleshooting, this is also failing in the containers on CircleCI with the following:

```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64
```

Basically, we have to get `fsevents` out of the `package.json` file because it's making the non-Mac build systems fail :)

**Comment by mattstratton** [comment]
Here's some more info why `fsevents` might be bailing

https://github.com/fsevents/fsevents/issues/301

**Comment by glasnt** [comment]
Oh right, I ran these commands on a mac, so that'd explain it.

**Comment by glasnt** [comment]
@mattstratton builds green :)

**Comment by mattstratton** [comment]
For a final test can you run `gulp dev` after removing `fsevents` to confirm it still works? Or do you already?

**Comment by glasnt** [comment]
@mattstratton Yes, it works, no more diffs in the code, with a local `gulp dev` and `hugo server -w`

**Pull request merged**